Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -373,11 +373,11 @@ def orbitRel : Setoid α where
 variable {G α}
 
 @[to_additive]
-theorem orbitRel_apply {a b : α} : (orbitRel G α).Rel a b ↔ a ∈ orbit G b :=
+theorem orbitRel_apply {a b : α} : orbitRel G α a b ↔ a ∈ orbit G b :=
   Iff.rfl
 
 @[to_additive]
-lemma orbitRel_r_apply {a b : α} : (orbitRel G _).r a b ↔ a ∈ orbit G b :=
+lemma orbitRel_r_apply {a b : α} : orbitRel G _ a b ↔ a ∈ orbit G b :=
   Iff.rfl
```

Comment:
These lemmas are now identical, so one of them should be deprecated.

Can you add a comment about the chosen naming convention to CoeFun instance?

Reply:
To whatever defines the coercion to function